src/Lab/Experiments/SilkBackedSkiaRendering/SilkBackedSkiaRendering.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net6.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+        <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\..\..\Windowing\Silk.NET.Windowing.Common\Silk.NET.Windowing.Common.csproj" />
+      <ProjectReference Include="..\..\..\Windowing\Silk.NET.Windowing.Glfw\Silk.NET.Windowing.Glfw.csproj" />
```

Comment:
Why are you pulling in just Glfw? Why not the Windowing metapacakge? (would eliminate the .Common reference as well)

Reply:
Because this only works with GLFW. Will see if I can also make it compatible with SDL

Reply:
There's no reason that I know why this wouldn't work with either, so give it a try :) if it doesn't work with both we should probably keep this in the Lab.